proxy/coco/src/project/checkout.rs

```diff
@@ -35,6 +37,22 @@ where
     include_path: PathBuf,
 }
 
+/// We want to know whether we're checking out from one of our own copies, or if we're checking out
+/// based off of a remote's branch.
+pub enum Ownership {
+    /// We're checking out our own copy of the project.
+    Local(PeerId),
+    /// We're checking out a remote's version of the project.
+    Remote {
+        /// The handle of the remote.
+        handle: String,
```

Comment:
Is this the handle as in `hash(id.revs[0])`?

Reply:
Nope, it's the handle of the peer. I'll update the docs :+1:

Reply:
Ok I'm not sure what that means, waiting to see the doc update :)

Reply:
Ah I see, it's the non-unique (display) name of the user.

Reply:
:100: good to know the docs helped :smile: